imports/api/playlists/server/publications.js

```diff
@@ -1,27 +1,32 @@
 import { Meteor } from 'meteor/meteor';
 import { Playlists } from '../playlists_collection.js';
-import { HTTP } from 'meteor/http';
-import { Random } from 'meteor/random';
 import { NowPlaying } from '../now_playing.js';
+import { pagination } from 'meteor/kurounin:pagination';
 
 Meteor.publish('playlist', function (id) {
   check(id, Number);
   return Playlists.find({ spinPlaylistId: id });
 });
 
-Meteor.publish('playlists', function(options) {
+Meteor.publish('playlists0', function(options) {
```

Comment:
I had to change it because the pagination package bases its functionality on the publication that has the same name as the collection.